### ISSUE TITLE
Template sanitizer: respect target

### ISSUE THREAD

**Issue by dvoytenko** [ampproject/amphtml#1572]

**Comment by dvoytenko** [comment]
Blocked by https://github.com/google/caja/issues/1991

**Comment by dvoytenko** [comment]
Workaround implemented. Still pursuing sanitizer fix.

**Comment by dvoytenko** [comment]
#2834 has been marked as a duplicate of this issue.

**Comment by Wadekone** [comment]
Per instruction from dvoytenko I would like to clarify if the solution for the issue was developed? 
Hope you understand this is an awful UX as every new page user opens is a new browser tab (way too much consumption of mobile device's resource at least). We are currently in a process of implementing the solution that will allow the customers to navigate through AMPs using the AMP CDN and this bug is causing a trouble. 
Hope someone can update this thread soon.

Thanks and Best Regards,
Vadim

**Comment by dvoytenko** [comment]
@Wadekone to provide the recap: I'm following up with the sanitizer library to address this issue.

**Comment by Wadekone** [comment]
@dvoytenko Appreciated, we are so excited to see the update!

**Comment by dvoytenko** [comment]
@Wadekone Once @5738 is merged, the `_top` target will be allowed and will also be default.

**Comment by Wadekone** [comment]
@dvoytenko  Thanks for the heads up! 
Side question if I may - is there any information on the amp availability by countries? I know there was a page with websites by countries [here](https://www.ampproject.org/) like 3 weeks ago, but now it's gone. Also, the last update on this was posted by Elena [here](http://searchengineland.com/google-amp-listings-feature-live-12-countries-246328) back in March. Many of our customers are wondering, perhaps you can put some light on this? 
Thanks!

**Comment by dvoytenko** [comment]
I'll refer to @rudygalfi to answer these questions.

**Comment by rudygalfi** [comment]
I think the page you're referring to was https://www.ampproject.org/learn/who, but it's been changed since. Are you asking specifically about where Google is driving traffic to AMP content (because of course there's nothing about AMP that's country specific).

In terms of countries, I think the article you linked is a majority. I don't have a list, but I think Australia, Canada, Argentina, Colombia have been added since. That's for the Top Stories feature. In terms of regular web results (https://blog.google/products/search/search-results-are-officially-ampd/), it's still rolling out globally.

**Comment by Wadekone** [comment]
@rudygalfi 
Rudy, thanks for your answer! My apologies for being slightly off-topic here.  
Basically, we've created AMP magento extension, and our clients are asking us, is AMP results are already being served in their countries. There's no strong knowledge of AMP among ecommerce stores owners yet, since there's only 1 major player in this area right now - the Ebay. But that's one of the main questions they usually ask - do they need this extension if their business is local? I wanted to make sure that I'm telling them precise info, and your answer is what I was looking for - the regular mobile search should be already serving users with AMPd pages. Thank you!

One other question while we're talking - Top Results section. Should we expect that the pages with type "article" will be included there, regardless if that's the news page, or store page? Appreciate your reply! 

By the way, the [WHO](https://www.ampproject.org/learn/who) page should also include Magento in the CMS section, since there are already 3 companies selling AMP extension for Magento ;) 

Thanks again for your time and for your useful responses!

Best Regards,

**Comment by rudygalfi** [comment]
With the Top Stories question, we're getting into Google Search behavior here, and this isn't really the right place for those questions, but [this](https://productforums.google.com/forum/#!topicsearchin/webmasters/category$3Aaccelerated-mobile-pages-amp%7Csort:relevance%7Cspell:false) is.

https://developers.google.com/search/docs/data-types/articles says: "Top Stories with AMP is a Google Search results feature that displays articles and video pages from sites" but the requirements are "the following schema.org types: Article, NewsArticle, BlogPosting, or VideoObject."
